let User,CustomErrorHandler;_6da‍.x([["default",()=>_6da‍.o]]);_6da‍.w("../../models",[["User",["User"],function(v){User=v}]]);_6da‍.w("../../services/CustomErrorHandler",[["default",["CustomErrorHandler"],function(v){CustomErrorHandler=v}]]);


const userController = {
  async me(req, res, next) {
    try {
      const user = await User.findOne({ _id: req.user._id }).select("-password -updatedAt -__v");

      if (!user) {
        return next(CustomErrorHandler.notFound());
      }

      res.json(user);
    } catch (error) {
      _6da‍.g.console.log(error);
      return next(error);
    }
  },
};

_6da‍.d(userController);
